third_party/graphlite/include/graph_lite.h

```diff
@@ -69,7 +69,7 @@ struct is_unordered_map<
 template <typename T>
 constexpr bool is_unordered_map_v = is_unordered_map<T>::value;
 template <typename T>
-constexpr bool is_either_map_v = is_map_v<T> or is_unordered_map_v<T>;
+constexpr bool is_either_map_v = is_map_v<T> || is_unordered_map_v<T>;
```

Comment:
I was considering making this a submodule, but maybe we should leave it as is so we can make these fixes...